packages/mongodb/_dev/deploy/docker/Dockerfile

```
@@ -0,0 +1,4 @@
+ARG SERVICE_VERSION=${SERVICE_VERSION:-4}
```

Please use a more specific tag, so we reduce the risk of unexpected changes when new minors of mongodb are released.
```suggestion
ARG SERVICE_VERSION=${SERVICE_VERSION:-4.4.8}
```

---

Ok got it
will do it.